PencilCase/PencilCase/www/index.html

```diff
@@ -238,6 +238,18 @@ <h1><span data-bind="text: title"></span></h1>
         <div data-role="header">
             <a href="#HomePage" data-icon="home" data-bind="click: navigator.goHome">Home</a>
             <h1><span data-bind="text: title"></span></h1>
+            <form>
+                <div class="ui-field-contain">
+                    <label for="title-filter-menu">Placeholder:</label>
+                    <select id="filtertext" data-native-menu="false" class="filterable-select">
+                        <option>Select fruit...</option>
+                        <option value="orange" data-filtertext="Florida">Orange</option>
```

Comment:
Note the data-filtertext field, which can be used to set filter text, this way makes the actually selected value and displayed text could be different field of one object.